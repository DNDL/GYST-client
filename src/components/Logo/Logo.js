import React from 'react';
import { Link } from 'react-router-dom';
import styles from './logo.css';

export default function Logo() {
  return (
    <section className={styles.logo}>
      <Link to={'/aboutus'}><h1>-GYST-</h1></Link>
    </section>
  );
}
