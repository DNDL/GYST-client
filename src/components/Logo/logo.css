@import url('https://fonts.googleapis.com/css?family=Averia+Serif+Libre|Dosis|Julius+Sans+One|Orbitron|Philosopher&display=swap');

@media only screen 
and (min-width: 360px) 
and (max-width: 600px) {
  .logo {
    margin: auto;
    text-align: center;
    width: 40vw;

    h1 {
      text-align: center;
      margin: 0;
      color: black;
      font-size: 2.5rem;
      font-family: 'Averia Serif Libre', cursive;

    }
  }
}
