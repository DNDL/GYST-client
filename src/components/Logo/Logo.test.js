import React from 'react';
import { shallow } from 'enzyme';
import Logo from './Logo';

describe('Logo component', () => {
  it('renders Logo', () => {
    const wrapper = shallow(<Logo />);
    expect(wrapper).toMatchSnapshot();
  });
});
