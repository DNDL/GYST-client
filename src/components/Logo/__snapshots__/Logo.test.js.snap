// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo component renders Logo 1`] = `
<Link
  to="/aboutus"
>
  <h1
    className="logo"
  >
    GYST
  </h1>
</Link>
`;
